src/controllers/donorController.ts

```diff
@@ -61,21 +65,99 @@ export const getDonations = (req: Request, res: Response) => {
  * Posts Donations
  * @route POST /donations
```

Comment:
Can you add a docstring about what the body should look like? Here's one way you could do it, although I'm not sure if it's the best.
```ts
@param {string} req.body.json Stringified JSON of type Donation
@param {UploadedFile?} req.files.descriptionImage
// etc.
```